### DISCUSSION TITLE
ESP32 aint getting recognize by the Pixelstick tool

### DISCUSSION THREAD

**Discussion by Modbusinternet** [forkineye/ESPixelStick#477] [General]
Hello, I am getting an error with esp32, I am trying to connect the esp32 d1 mini to the espsticktool to upload but ain't getting recognized, saying failed to connect. 
It worked a few days ago but now all my esp32 are stuck in the same problem. However, my esp8266 is working correctly.

**Comment by MartinMueller2003** [comment]
Can you give me a bit more information about which version of the tools you are using?
